packages/notification-managment-spa/src/app/form/form.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { AppService } from '../app.service';
+import { UserProfile } from '../helper';
+
+@Component({
+  selector: 'op-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.scss']
+})
+export class FormComponent implements OnInit {
+  user = UserProfile;
+  days: Day[] = [
```

Comment:
You can automate the day population by moment. please see this https://stackoverflow.com/questions/25905183/using-moment-js-to-create-an-array-with-days-of-the-week-and-hours-of-the-day

Reply:
we don't wanna use the swiss army knife to cut a cake... it's pretty much simple, readable and easily created with a simple JSON data don't need to depend on 3rd party library to generate. I think after the backend is ready to use then it might useful but for now, I don't think we need to use big guns.